api/k8sdeps/kunstruct/factory.go

```diff
@@ -116,6 +118,21 @@ func (kf *KunstructuredFactoryImpl) validate(u unstructured.Unstructured) error
 	return nil
 }
 
+// skipKustomization if set on a Resource will skip kustomizing the Resource
+const skipKustomization = "config.kubernetes.io/local-config"
```

Comment:
Since we are consuming raw config directories, we need some way to prevent accumulating and applying non-resource configuration files -- e.g. like kustomization.yaml, but for other tools

Reply:
i'll create the separate PR

Reply:
There could be resources that kustomize should read and pass thru (unkustomized) and perhaps some that kustomize should ignore completely (things one might put in a `.ignore`).

So the constant name and comment should be clear on which one.  I think it would be clearer for release notes, review, testing to do this in a distinct PR.

Reply:
#1709